src/main_thread/init/utils/create_core_playback_observer.ts

```diff
@@ -149,6 +170,7 @@ export default function createCorePlaybackObserver(
         },
         readyState: observation.readyState,
         speed: lastSpeed,
+        canStream: mediaSource?.streaming,
```

Comment:
Why not defaulting to `true` or `false` when not defined?

Reply:
Done, default it to true